app/src/main/res/layout/home_fragment.xml

```diff
@@ -16,19 +16,35 @@
     android:background="@color/white"
     android:orientation="vertical">
 
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:background="@color/white"
+      android:gravity="center"
+      android:orientation="vertical">
+
+      <Button
+        android:id="@+id/play_exploration_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:onClick="@{presenter::playExplorationButton}"
+        android:text="Play exploration"
+        android:visibility="gone" />
+
+      <Button
+        android:id="@+id/topic_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:onClick="@{presenter::handleTopicButtonClicked}"
+        android:text="Go to topic" />
```

Comment:
Why do we need this? We should load topics based on the recycler view. Let's not add anymore temporary buttons in the home screen--we need to rely on existing UIs at this point.

Reply:
Considering this, we might need to change the testing strategy meaning, creating a new TestActivity which loads TopicFragment for a specific TOPIC_ID.

Reply:
Yes, that's what we should be doing. That also future proofs this as we start loading real data in the domain layer since we can always load a test topic ID for the purpose of these tests.

Reply:
@nikitamarysolomanpvt since this requires downstream test changes, I think it's fine to do this in a later PR as part of resolving #135.

Reply:
Okay will create separate PR for this